pages/common/tree.md

```diff
@@ -14,14 +14,14 @@
 
 `tree -a`
 
-- Print human readable size of files:
+- Print the tree without indentation lines, showing the full path instead:
 
-`tree -h`
+`tree -i -f`
 
-- Print the full path for each file:
+- Print the size of each node next to it, in human-readable format, with folders displaying their cumulative size (as in the `du` command):
 
-`tree -f`
+`tree -s -h --du`
 
-- Print the tree without lines and indentation. Useful when used with -f:
+- Filter the tree using a wildcard (glob) pattern, showing only the hierarchies leading to the matching entries:
 
-`tree -i`
+`tree -P {{*.txt}} --prune`
```

Comment:
Ah, this is a good one. I typically end up `tree -a | grep -v`-ing out the things I don't want to see.

I'd stick with the simple and just use `-P {{pattern}}` and let people figure out the rest.

`--prune` seems excessive. Why filter out empty directories in this general an example? They won't hurt if `--matchdirs` isn't set as they contain nothing to match on.

`--matchdirs` is probably relevant to this pattern matching sample, but makes the command more daunting to deal with at first.

In my book, `-a` means `anarchic`. But most likely you're not trying to look for things inside `.git`. If you are, and you know of this folder, you likely know it's a _hidden_ folder and that it'll be ignored by default by most commands that follow this convention. 

I'd stick with the simple.